### PR TITLE
Adding asset tracking telemetry testing

### PR DESCRIPTION
**Pull request by aburhan**

### CODE REVIEW

File: outer-loop-cli/src/devai/commands/gitlab.py [hunk 1]
```diff
@@ -18,8 +18,11 @@
 from langchain_google_vertexai import ChatVertexAI
 from langchain_community.agent_toolkits.gitlab.toolkit import GitLabToolkit
 from langchain_community.utilities.gitlab import GitLabAPIWrapper
+  
+USER_AGENT = 'cloud-solutions/genai-outer-loop-devai'
 
-llm = ChatVertexAI(model_name="gemini-pro",
+with telemetry.tool_context_manager(USER_AGENT):
```
Comment (gitrey) [inline outer-loop-cli/src/devai/commands/gitlab.py:25]: @aburhan do we need to import anything for `telemetry.tool_content_manager`?

Comment (aburhan) [reply outer-loop-cli/src/devai/commands/gitlab.py:25]: updated

File: outer-loop-cli/src/devai/commands/gitlab.py [hunk 2]
```diff
@@ -18,8 +18,12 @@
 from langchain_google_vertexai import ChatVertexAI
 from langchain_community.agent_toolkits.gitlab.toolkit import GitLabToolkit
 from langchain_community.utilities.gitlab import GitLabAPIWrapper
+from google.cloud.aiplatform import telemetry
+  
+USER_AGENT = 'cloud-solutions/genai-outer-loop-devai'
```
Comment (gitrey) [inline outer-loop-cli/src/devai/commands/gitlab.py:23]: I will merge and change this to 'cloud-solutions/genai-for-developers-v1'

**Review by gitrey — APPROVED** [review]
LGTM

**Pull request merged**